model/service_logs/v1/cluster_log_type.model

```diff
@@ -66,4 +66,10 @@ class LogEntry {
 
 	// Type of the service log entry.
 	LogType LogType
+
+    // The time at which the cluster log was created
+	CreatedAt Date
+
+    // The name of the user who created the cluster log
```

Comment:
nit: comment whitespace does not match the tabbing of the other attributes

Reply:
good catch! fixed.